cli/lib/webpack/service.js

```diff
@@ -0,0 +1,226 @@
+const assert = require('assert');
+const crypto = require('crypto');
+const EventEmitter = require('events');
+
+const WebpackProgressReporter = require('./reporter');
+
+class WebpackService {
+	constructor(options) {
+		const { AppcdClient, builder, cli, logger } = options;
```

Comment:
I think you could put this destructuring into the function param declaration: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_function_parameter